PP_APP/server/pp_api/models.py

```diff
@@ -13,9 +13,22 @@ def upload_top_location(instance, filename):
 def upload_side_location(instance, filename):
     return 'side/{author_id}/{filename}'.format(author_id=str(instance.user_id.id), filename=filename)
 
+def upload_infer_location(instance, filename):
+    return 'infer/{author_id}/{filename}'.format(author_id=str(instance.user_id.id), filename=filename)
+
 class image_post(models.Model):
     user_id = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
-    packager = models.CharField(max_length=30, blank=True)
-    score = models.DecimalField(max_digits=2, decimal_places=1,blank=True, null=True)
     top_img = models.ImageField(upload_to=upload_top_location, null=False,blank=False)
-    side_img = models.ImageField(upload_to=upload_side_location, null=False,blank=False)
\ No newline at end of file
+    side_img = models.ImageField(upload_to=upload_side_location, null=False,blank=False)
+
+    def create(self, validated_data):
+        return image_post(**validated_data)
+
+class image_post_predicted(models.Model):
```

Comment:
I only have a comment about naming; since a few of us are gonna be touching Django, naming and docs is important. 

I understand that this model concerns the mask-rcnn evaluation but I'd make this really obvious by putting it in the docstring.